ql/src/java/org/apache/hadoop/hive/ql/security/authorization/HiveCustomStorageHandlerUtils.java

```diff
@@ -90,4 +92,21 @@ public static boolean getWriteOperationIsSorted(Configuration conf, String table
         String isSortedString = conf.get(WRITE_OPERATION_IS_SORTED + tableName);
         return Boolean.parseBoolean(isSortedString);
     }
+
+    public static void setWriteOperationIsMerge(Configuration conf, String tableName, boolean isMerge) {
+        if (conf == null || tableName == null) {
+            return;
+        }
+
+        conf.set(WRITE_OPERATION_FOR_MERGE_TASK + tableName, Boolean.toString(isMerge));
+    }
+
+    public static boolean getWriteOperationIsMerge(Configuration conf, String tableName) {
```

Comment:
why not call it 'isMergeTask' for clarity?

Reply:
Done.